trane/datasets/load_functions.py

```diff
@@ -98,6 +98,17 @@ def load_youtube_metadata():
     return table_meta
 
 
+def load_airbnb_reviews():
+    time_col = "date"
+    filepath = generate_local_filepath("data/airbnb_reviews/airbnb_reviews.csv.bz2")
+    df = pd.read_csv(filepath, dtype_backend="pyarrow")
+    df = df.dropna()
+    df[time_col] = pd.to_datetime(df[time_col], format="%Y-%m-%d")
+    df = df.sort_values(by=["date"])
```

Comment:
Created an issue to have Trane auto-do this:
https://github.com/trane-dev/Trane/issues/126